airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -1042,7 +1042,7 @@ public boolean shouldRunConnectionManagerWorkflows() {
 
   @Override
   public boolean shouldRunNotifyWorkflows() {
-    return getEnvOrDefault(SHOULD_RUN_NOTIFY_WORKFLOWS, false);
+    return getEnvOrDefault(SHOULD_RUN_NOTIFY_WORKFLOWS, true);
```

Comment:
Is there anything else in the env that needs to be configured in order to be able to make a notification. If so we should leave the default as false.

Reply:
I'm not sure - that's why I set the default to true

Reply:
So should the default be false then so we don't try to send a notification if the webhook is not configured in an OSS deployment?

Reply:
all of the other workflow env variables are defaulting to true so I thought I'd follow the same pattern - is there a reason not to?

Reply:
We should not be using this anymore.  Instead, we should be using configuration injected by Micronaut.  If we need to change the default to true, we should do so in the `application.yaml` files for the appropriate environment.

Reply:
so all of the workflows should default to false?

Reply:
talked to @jdpgrailsdev separately - this was about where we're setting the env variable. I'm setting it to true in all places so we don't have potential conflicts. @benmoriceau is it ok to leave this to true like other workflows? users have a separate setting for turning off these notifications. I'm worried it might be confusing if this is the only workflow that defaults to false

Reply:
The notification is in the payload, so it should be ok to set it as true. I think that generally it is safer to have an opt-in strategy in OSS rather until we are confident that something is working after having it live in cloud for a little bit.